src/backends/backend/tensorflow/tensorflow.cc

```diff
@@ -1849,6 +2149,60 @@ TRITONBACKEND_Initialize(TRITONBACKEND_Backend* backend)
         "triton backend API version does not support this backend");
   }
 
+  // The backend configuration may contain information needed by the
+  // backend, such a command-line arguments. This backend doesn't use
+  // any such configuration but we print whatever is available.
+  TRITONSERVER_Message* backend_config_message;
+  RETURN_IF_ERROR(
+      TRITONBACKEND_BackendConfig(backend, &backend_config_message));
+
+  const char* buffer;
+  size_t byte_size;
+  RETURN_IF_ERROR(TRITONSERVER_MessageSerializeToJson(
+      backend_config_message, &buffer, &byte_size));
+  LOG_MESSAGE(
+      TRITONSERVER_LOG_INFO,
+      (std::string("backend configuration:\n") + buffer).c_str());
+
+  ni::TritonJson::Value backend_config;
+  if (byte_size != 0) {
+    RETURN_IF_ERROR(backend_config.Parse(buffer, byte_size));
+  }
+
+  // Set default backend values. Note that those values should be set during
+  // TRITONBACKEND_Initialize, but for TensorFlow they are set while creating
```

Comment:
I don't understand this comment... we are setting the values and we are in TRITONBACKEND_Initialize